eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -347,14 +344,28 @@ stages:
             demands: ${{ variables.defaultPoolDemandsWindows }}
           container: ''
           excludeOmniSharpTests: true        # ✅
-          targetRid: 'win-x64'               # 📝
+          targetRid: 'win-x64'               # 📝 # TODO: remove this once https://github.com/dotnet/installer/pull/18476 lands 
 
       ### Additional jobs for full build ###
       - ${{ if in(parameters.scope, 'full') }}:
 
         - template: ../jobs/vmr-build.yml
           parameters:
-            buildName: MarinerCrossArm64
+            buildName: MarinerCross
```

Comment:
AFAIK the cross build are currently broken. Shouldn't they be commented so that full builds are green? cc @directhex